### ISSUE TITLE
The link for CIFAR100 32x32 ViT pretrained weight is actually the CIFAR 10 ViT

### ISSUE THREAD

**Issue by yyyyychen** [hananshafi/vits-for-small-scale-datasets#4]

**Comment by yyyyychen** [comment]
I am quite interested in your wonderful work. This is a meaningful work to help understanding the transformer networks on datasets of different scales.
However, when using finetune.py to evaluate the accuracy of the pretrained weights swin_cifar10, swin_cifar100, vit_cifar10, I get almost random classification results.
Would you give me some help how to evaluate the pretrained weights correctly, thank you!

**Comment by hananshafi** [comment]
Hello, you must download the weights of each model from our repo.
